geopy/geocoders/googlev3.py

```diff
@@ -97,10 +100,12 @@ def __init__(
             self.premier = True
```

Comment:
That looks better than the code here. I just didn't want to refactor it and only modify the minimum necessary so it could move through faster.

Since we are maintaining this fork, I will take your suggestion.